Documentation: Update documentation for Infineon boards

mcatee-infineon opened the pull request:
Documentation: Update documentation for Infineon boards

	-Update formatting and contents of index.rst for cy8ckit_062s4
	-Update formatting and contents of index.rst for cy8ckit_064s0s2_4343w
	-Update formatting and contents of index.rst for cy8cproto_062_4343w
	-Update formatting and contents of index.rst for cy8cproto_063_ble
	-Update formatting and contents of index.rst for xmc45_relax_kit
	-Update formatting and contents of index.rst for xmc47_relax_kit
	-Change all instances of "PSoC" to "PSOC" for infineon platforms

kartben commented:
I would suggest not touching the past release notes, if that's ok with you. Unless *really* necessary I think it's best to try and keep these files untouched as they're by definition meant to be reflecting how things were at release time

mcatee-infineon committed:
Documentation: Update documenation for Infineon boards

	-Update formatting and contents of index.rst for cy8ckit_062s4
	-Update formatting and contents of index.rst for cy8ckit_064s0s2_4343w
	-Update formatting and contents of index.rst for cy8cproto_062_4343w
	-Update formatting and contents of index.rst for cy8cproto_063_ble
	-Update formatting and contents of index.rst for xmc45_relax_kit
	-Update formatting and contents of index.rst for xmc47_relax_kit
	-Change all instances of "PSoC" to "PSOC" for infineon platforms

Signed-off-by: McAtee Maxwell <maxwell.mcatee@infineon.com>

mcatee-infineon commented:
> I would suggest not touching the past release notes, if that's ok with you. Unless _really_ necessary I think it's best to try and keep these files untouched as they're by definition meant to be reflecting how things were at release time. This is actually why these files are not compliant from a .rst perspective as I didn't want to "touch" them just for the sake of making them compliant when I introduced the ReST linter

@kartben, that makes total sense. I have changed the PR to reflect this. Thank you for your comment.

dkalowsk commented on boards/infineon/cy8cproto_062_4343w/doc/index.rst:
```diff
@@ -101,50 +102,58 @@ To fetch Binary Blobs:
 
-OpenOCD Installation
-====================
+The CY8CPROTO-062-4343W includes an onboard programmer/debugger (`KitProg3`_) to provide debugging, flash programming, and serial communication over USB. Flash and debug commands use OpenOCD and require a custom Infineon OpenOCD version, that supports KitProg3, to be installed.
```
Previously these lines were wrapped at 80 columns now they seem a little longer.  Are you sure that is what you want?  Commenting on this only once as it spans a few different diff sections.

dkalowsk reviewed (commented):
This is a rather large PR with a single commit.  I'd really recommend breaking it up a bit for easier review.  For example commit 1 being just the name change.  Commits N being the new text being added.  Mostly a suggestion for the next round.

Anyways after having read through it minor comment on formatting.  The text reads clearly and I believe all the formatting is correct.

mmahadevan108 commented:
Please rebase and push again to restart CI

jhedberg commented:
> Please rebase and push again to restart CI

No need to rebase since there aren't any conflicts. I triggered a rerun of the failed CI Checks manually.

mmahadevan108 commented:
@mcatee-infineon , can you help address the CI failures.

mmahadevan108 commented:
Ping @mcatee-infineon , can you take a look at the CI failures

dkalowsk commented:
@mcatee-infineon, @ifyall, @npal-cy, @talih0 as the creator of the PR and maintainer/collaborators for Infineon, can one of you please fix up this PR for the 4.0 final release happening at the end of the week?

npal-cy commented:
let me check this...
